### ISSUE TITLE
Submit a PR for leela zero instead of own patch

### ISSUE THREAD

**Issue by bood** [featurecat/lizzie#39]
Why not submit a PR and add a switch to enable the continuous dump stats? so Lizzie can be used with the original leela zero

I can help if you want

**Comment by featurecat** [comment]
Yes, please help! It should be pretty simple... some command like --ponder-dumpstats-frequency which dumps stats from ponder (without "~begin" and "~end" like I have), every $ponder-dumpstats-frequency$ milliseconds. I am busy once again so I can't develop full time. Any contributions are greatly appreciated :)

**Comment by dwt** [comment]
How about this to get an upstream merging discussion started?

**Comment by featurecat** [comment]
copying my post to your pr for convenience...

Have you seen gcp#1242? Someone tried to submit a PR recently to GCP's Leela Zero but he denied the changes because it's not threadsafe. He said he didn't like the idea of a command line switch but that it should be a GTP command... but he didn't like Cabuu's implementation. He said he will do it himself, so I think for now we should wait on that.

I don't know when exactly he will make these changes.
